### ISSUE TITLE
there is no extra in geometry nodes add menu！！！

### ISSUE THREAD

**Issue by Darkoxen** [SpectralVectors/MindMapper#11]
There is no extra in geometry nodes add menu！！！ 
test with both 3.5.1 3.6.5
![Snipaste_2023-10-30_21-15-35](https://github.com/SpectralVectors/MindMapper/assets/11469635/f1b02880-5b99-4309-b0f7-0054d30395ac)

**Comment by SpectralVectors** [comment]
Yes, this is a known issue, I should have documented it on the project homepage.

The nodes are there if you Search for them, but I'm not sure why the 'Extra' menu doesn't display in Geometry Nodes.
I will take another look to see if I can figure out why it's not displaying, until then you will have to use Search to access them.

Searching 'mind' or 'big' will find the main nodes, apologies for the workaround!
